src/main/java/com/minecolonies/coremod/client/gui/WindowTownHall.java

```diff
@@ -994,18 +1024,24 @@ public void onUpdate()
         super.onUpdate();
 
         final String currentPage = findPaneOfTypeByID(VIEW_PAGES, SwitchView.class).getCurrentView().getID();
-        if (currentPage.equals(PAGE_PERMISSIONS))
-        {
-            updateUsers();
-            window.findPaneOfTypeByID(LIST_USERS, ScrollingList.class).refreshElementPanes();
-        }
-        else if (currentPage.equals(PAGE_CITIZENS))
+        switch (currentPage)
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a default case to this switch. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3ASwitchLastCaseIsDefaultCheck)